blueocean-web/src/main/js/main.jsx

```diff
@@ -26,7 +26,7 @@ class App extends Component {
             <div className="Site">
                 <div id="outer">
                     <header className="global-header">
-                        <ExtensionPoint name="jenkins.logo.top"/>
+                        <Extensions.Renderer name="jenkins.logo.top"/>
```

Comment:
... maybe you'd prefer the import just referred to the library as `ExtensionPoint`? And you'd have an `ExtensionPoint.Renderer` as the component to render them?

```javascript
import ExtensionPoint from '@jenkins-cd/js-extensions';
...
<ExtensionPoint.Renderer name="jenkins.logo.top" ... />
```

I'm sort of indifferent about this detail. Except for the `name` property which is inconsistent with `extensionPoint` in the yaml file.